linux_os/guide/system/software/updating/ensure_gpgcheck_globally_activated/rule.yml

```diff
@@ -86,8 +86,11 @@ platform: yum
 
 fixtext: |-
     Configure {{{ full_name }}} to always check package signatures before installation.
-    Add or update the following line in {{{ pkg_manager_config_file }}}:
+
+    Add or update the following line in [main] section of the {{{ pkg_manager_config_file }}} file:
+
     gpgcheck=1
 
 srg_requirement: |-
-   {{{ full_name }}} must have GPG checking enabled in the {{{ pkg_manager }}} package manager.
+    {{{ full_name }}} must prevent the installation of software, patches, service packs, device drivers, or operating system components from a repository without verification they have been digitally signed using a certificate that is issued by a Certificate Authority (CA) that is recognized and approved by the organization.
+    {{{ full_name }}} must have GPG checking enabled in the {{{ pkg_manager }}} package manager.
```

Comment:
Are you sure that srg_requirement should have two entries like this?

Reply:
it shouldn't be there.